src/vclock.rs

```diff
@@ -32,7 +32,7 @@ use crate::{Actor, Causal, CmRDT, CvRDT, Dot};
 /// It can tell you if something causally descends something else,
 /// or if different replicas are "concurrent" (were mutated in
 /// isolation, and need to be resolved externally).
-#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, Ord)]
```

Comment:
Ah wait I don't think we can do this in general, {A:1} is not comparable to {B:2}. We only get a partial order over VClocks.